index.html

```diff
@@ -171,7 +171,7 @@ <h2 class="title-card-global">
     <div class="lang-content">
       <div class="img-globe">
           <div class="container-img">
-              <img src="images/Group.png" id="globe-img" alt="skills icon">
```

Comment:
- [ ] You have done a great job implementing accessibility in your portfolio :muscle: but there is an issue with the design of the buttons because these are not passing the contrast check. I would suggest fixing this contrast issue for better accessibility :wink:
**You can see the [contrast test results here](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2021-11-09-at-20-37-14-560?failing)**

![image](https://user-images.githubusercontent.com/81522435/141001415-9d1f2121-c094-4b90-9677-107cd4929045.png)

![image](https://user-images.githubusercontent.com/81522435/141001457-9949a768-6e12-4450-875c-e6bfd315fcad.png)

![image](https://user-images.githubusercontent.com/81522435/141001520-4cbea673-2676-4d6d-a71d-8e958536b328.png)

- [ ] [OPTIONAL] I think there is an issue with the mobile version because some of the titles are aligned to the left or are taking the whole width. It could be a good idea to fix this so your portfolio looks even better :fire:

  ![image](https://user-images.githubusercontent.com/81522435/141002008-6f8b31a4-c122-46f1-be6a-f2d023128d96.png)